### DISCUSSION TITLE
Conversation stops

### DISCUSSION THREAD

**Discussion by richard-deetlefs** [microsoft/autogen#226] [Q&A]
Why does the conversation stop at some point and exit? The script just stops running at some point, after I either provide feedback or press enter to skip and use auto-reply. I do not type 'exit'.

**Comment by sonichi** [comment]
https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#__init__
Either max consecutive auto replies is reached, or the message is a termination message.
